mscore/shortcut.cpp

```diff
@@ -640,6 +703,17 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
+         "reset-groupings",
+         QT_TRANSLATE_NOOP("action","Reset Note and Rest Groupings"),
+         QT_TRANSLATE_NOOP("action","Reset note and rest groupings"),
```

Comment:
Thanks. I will change it to "Regroup rhythms" or whatever it is in the master branch.